src/Pages/Article.tsx

```diff
@@ -55,29 +57,34 @@ const Article = ({ language, match: { params: { articleId } }, t }) => {
         const currentLanguage = language || defaultLanguage;
 
         spinnerService.hide("apiSpinner");
-        setArticle(data => ({
+        setArticle((data: any) => ({
           ...data,
           [currentLanguage]: response.data.items[0]
         }));
       });
   }, [language, articleId]);
 
   const currentArticle = article[language];
-  if (currentArticle && currentArticle.length === 0) {
+  if (!currentArticle) {
     return <div className="container" />;
   }
 
-  let formatDate = value => {
-    return dateFormat(value, 'dddd, mmmm d, yyyy');
+  let makeFormatDate = (value: string): string => {
```

Comment:
All f these lets can be const

Reply:
[60c8df1](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/60c8df181a955593d765a8743f6679de232737f5)